frontend_app_gallery/src/container/Gallery.jsx

```diff
@@ -0,0 +1,485 @@
+      )
+    }
+
+    switch (fetchContentList.apiResponse.status) {
+      case 200:
+        let images = []
+        fetchContentList.body.forEach(content => {
+          if (content.content_type === 'file') {
+            images.push({ src: '', contentId: content.content_id })
+          }
+        })
+        const imagesPreviews = await this.loadPreview(images)
+
+        this.setState({ imagesPreviews })
+
+        break
+      default: this.sendGlobalFlashMessage(props.t('Error while loading content list'))
+    }
+  }
+
+  loadPreview = async (images) => {
+    const { state, props } = this
+
+    return Promise.all(images.map(async (image) => {
+      const fetchFileContent = await handleFetchResult(
+        await getFileContent(state.config.apiUrl, state.config.appConfig.workspaceId, image.contentId)
+      )
+      switch (fetchFileContent.apiResponse.status) {
+        case 200:
+          const filenameNoExtension = removeExtensionOfFilename(fetchFileContent.body.filename)
+          const previewUrl = buildFilePreviewUrl(state.config.apiUrl, state.config.appConfig.workspaceId, image.contentId, fetchFileContent.body.current_revision_id, filenameNoExtension, 1, 1400, 1400)
+          const previewUrlForThumbnail = buildFilePreviewUrl(state.config.apiUrl, state.config.appConfig.workspaceId, image.contentId, fetchFileContent.body.current_revision_id, filenameNoExtension, 1, 150, 150)
+          const lightBoxUrlList = (new Array(fetchFileContent.body.page_nb)).fill('').map((n, j) =>
+            buildFilePreviewUrl(state.config.apiUrl, state.config.appConfig.workspaceId, image.contentId, fetchFileContent.body.current_revision_id, filenameNoExtension, j + 1, 1920, 1920)
+          )
+
+          return {
+            ...image,
+            src: previewUrl,
+            fileName: fetchFileContent.body.filename,
+            lightBoxUrlList,
+            previewUrlForThumbnail,
+            rotationAngle: 0
+          }
+        default:
+          this.sendGlobalFlashMessage(props.t('Error while loading file preview'))
+          return {}
+      }
+    }))
+  }
+
+  loadWorkspaceData = async () => {
+    const { state, props } = this
+
+    const fetchResultWorkspaceDetail = await handleFetchResult(
+      await getWorkspaceDetail(state.config.apiUrl, state.config.appConfig.workspaceId)
+    )
+
+    switch (fetchResultWorkspaceDetail.apiResponse.status) {
+      case 200:
+        this.setState({
+          content: {
+            workspaceLabel: fetchResultWorkspaceDetail.body.label
+          }
+        })
+        break
+      default:
+        this.sendGlobalFlashMessage(props.t('Error while loading shared space detail'))
+    }
+  }
+
+  sendGlobalFlashMessage = msg => GLOBAL_dispatchEvent({
+    type: CUSTOM_EVENT.ADD_FLASH_MSG,
+    data: {
+      msg: msg,
+      type: 'warning',
+      delay: undefined
+    }
+  })
+
+  handleClickHideImageRaw = () => {
+    this.setState({ displayLightbox: false, fullscreen: false })
```

Comment:
fullscreen and fileSelected got modified only on one case (show or hide).
fileSelected should be reset to a default value when hidden and, from here, I don't think fullscreen is useful

Reply:
`fullscreen` is useful because when the fullscreen is enabled, the lightbox is rendered fullscreen and the close button of the lightbox is still visible, so when this button is clicked the lightbox need to hide itself and disable fullscreen in the case of fullscreen enabled